src/test/java/jp/co/soramitsu/sora/crypto/algorithms/TestVectors.java

```diff
@@ -0,0 +1,60 @@
+package jp.co.soramitsu.sora.crypto.algorithms;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import jp.co.soramitsu.crypto.ed25519.Utils;
+
+public class TestVectors {
```

Comment:
I suggest to change this class to arguments provider (see @ArgumentsSource) and use in tests annotation @ParameterizedTest